core/templates/pages/admin-page/navbar/admin-navbar.directive.html

```diff
@@ -174,9 +174,6 @@
                   <div ng-if="!$ctrl.profilePictureDataUrl" class="dropdown-toggle">
                     <i class="material-icons md-40 oppia-dropdown-toggle-icon">&#xE853;</i>
                   </div>
-                  <div class="oppia-user-email oppia-user-email-text">
```

Comment:
Do we still need the CSS code for `oppia-user-email` and `oppia-user-email-text`?

Reply:
Removed the `oppia-user-email-text` but couldn't find `oppia-user-email`

Reply:
It doesn't seem to be defined anywhere.